### ISSUE TITLE
Angular CLI Crashes when executing NG G D

### ISSUE THREAD

**Issue by briamib** [angular/angular-cli#23693]
# 🐞 Bug report

### Command (mark with an `x`)

- [ X] new
- [ ] build
- [ ] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] extract-i18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc

### Is this a regression?

 Yes, the previous version in which this bug was not present was: ....
Yes the last version of angular 14.0.0
### Description

 A clear and concise description of the problem...

## 🔬 Minimal Reproduction

by executing the command ng g d to generate a directive 

You can read more about issue submission guidelines here: https://github.com/angular/angular-cli/blob/main/CONTRIBUTING.md#-submitting-an-issue
-->

## 🔥 Exception or Error

<pre><code>

catch clause variable is not an Error instance
See "/private/var/folders/9q/zm232ccd4hx209x2zklqt5mc0000gn/T/ng-MHxcXl/angular-errors.log" for further details
</code></pre>

## 🌍 Your Environment

<pre><code>

Angular CLI: 14.1.0
Node: 16.14.0
Package Manager: npm 8.11.0 
OS: darwin arm64

Angular: 14.1.0
... animations, cli, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, platform-server
... router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1400.6
@angular-devkit/build-angular   14.1.0
@angular-devkit/core            14.0.6
@angular-devkit/schematics      14.1.0
@nguniversal/builders           14.0.3
@nguniversal/express-engine     14.0.3
@schematics/angular             14.1.0
rxjs                            7.5.6
typescript                      4.7.4
</code></pre>

**Anything else relevant?**

the log file created
```
[error] AssertionError [ERR_ASSERTION]: catch clause variable is not an Error instance
    at assertIsError (/Users/ME/development/platform/cacheclient/node_modules/@angular/cli/src/utilities/error.js:16:26)
    at GenerateCommandModule.runSchematic (/Users/ME/development/platform/cacheclient/node_modules/@angular/cli/src/command-builder/schematics-command-module.js:311:43)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async GenerateCommandModule.handler (/Users/ME/development/platform/cacheclient/node_modules/@angular/cli/src/command-builder/command-module.js:109:24)
```

**Comment by alan-agius4** [comment]
Thanks for reporting this issue. Luckily, it has already been fixed in one of the recent releases. Please update to the most recent version to resolve the problem.

If the problem persists in your application after upgrading, please open a new issue, provide a simple repository reproducing the problem, and describe the difference between the expected and current behavior. You can use `ng new repro-app` to create a new project where you reproduce the problem.

**Comment by briamib** [comment]
it hasn't been fixed - i am on the latest version.  closing tickets just to close them is not acceptable.

Thanks in advance…

Chris Burd
Maxus Capital Group, LLC
________________________________

**Comment by briamib** [comment]
> Thanks for reporting this issue. Luckily, it has already been fixed in one of the recent releases. Please update to the most recent version to resolve the problem.
> 
> If the problem persists in your application after upgrading, please open a new issue, provide a simple repository reproducing the problem, and describe the difference between the expected and current behavior. You can use `ng new repro-app` to create a new project where you reproduce the problem.

IF YOU HAD FIXED IT I WOULDN'T HAVE OPENED A TICKET 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡

**Comment by briamib** [comment]
> Thanks for reporting this issue. Luckily, it has already been fixed in one of the recent releases. Please update to the most recent version to resolve the problem.
> 
> If the problem persists in your application after upgrading, please open a new issue, provide a simple repository reproducing the problem, and describe the difference between the expected and current behavior. You can use `ng new repro-app` to create a new project where you reproduce the problem.

moreover - I love your thinking - toss months of work in the trash because you guys can't get your sh#t together.  This is more and more why we are looking at alternative frameworks.

**Comment by alan-agius4** [comment]
Based on the output of `ng version` you are not on the latest version.  Please update to the latest version.

Please refrain from using foul language and follow our [code of conduct](https://github.com/angular/angular-cli/blob/main/CODE_OF_CONDUCT.md).

**Comment by briamib** [comment]
you would be well remised to remind yourself it is the open source community that gives you a job - instead of dismissing so readily, and being so damn rude all the time.

Angular has a long history of bugs - some of which have been sitting for years.  the truth is if you had done QA on all versions (properly) and tested each feature, service and directive instead of trying to sprint to GIGO software - we wouldn't have had many people not just I sitting idle - so take that internal.

At this juncture bullying me about what I said - and your shear rudeness is unacceptable.  The word shit - and your attitude go hand in hand.


Chris Burd
Maxus Capital Group, LLC
________________________________

**Comment by mgechev** [comment]
Hey Chris, as @alan-agius4 mentioned, we fixed this issue in v14.1.1. Make sure you update to the latest patch version to resolve the problem. I'll lock the conversation because it seems to be going off topic.

In case update to 14.1.1 does not help, please open a new issue with a reproduction so we can investigate.

Thank you!
